### PR TITLE
fix default icons on iOS

### PR DESCRIPTION
**Pull request by joshblour**
This fixes a crash on iOS when no icon is set (want to use default red icon)

### CODE REVIEW

File: PPTGoogleMap.js [hunk 1]
```diff
@@ -90,12 +90,15 @@ class MapView extends Component {
...
-            this._markersForBridge.push({
+            let markerProps = {
                 publicId: marker.id,
                 latitude: marker.latitude,
-                longitude: marker.longitude,
-                icon: resolveAssetSource(marker.icon),
-            });
+                longitude: marker.longitude,  
+            }
+            if (marker.icon)
```
Comment (edcs) [inline PPTGoogleMap.js]: Please could you add `{}` braces into this `if` statement.

File: ios/PPTGoogleMap.m [hunk 2]
```diff
@@ -102,7 +102,9 @@ - (void)setMarkers:(NSArray *)markers
         
         GMSMarker* mapMarker = [GMSMarker markerWithPosition:CLLocationCoordinate2DMake(latitude, longitude)];
         
-        mapMarker.icon = [self getMarkerImage:marker];
+        if (marker[@"icon"])
```
Comment (edcs) [inline ios/PPTGoogleMap.m]: Please could you add `{}` braces into this `if` statement.

**Comment by joshblour** [comment]
sure thing! done

File: ios/PPTGoogleMap.m (hunk 2 above)
Comment (edcs) [inline ios/PPTGoogleMap.m]: Not sure if you saw this one, but this if needs braces too.

**Comment by joshblour** [comment]
ok, done.

**Pull request merged**